packages/wrangler/src/api/startDevWorker/RemoteRuntimeController.ts

```diff
@@ -339,6 +371,7 @@ export class RemoteRuntimeController extends RuntimeController {
 		logger.debug("RemoteRuntimeController teardown beginning...");
 		this.#session = undefined;
 		this.#abortController.abort();
+		this.#activeTail?.terminate();
```

Comment:
I assume it is safe to terminate (rather than somehow cancel the tailing) as the remote worker itself is about to be killed anyway.